pkg/assembler/backends/neo4j/pkg.go

```diff
@@ -699,3 +699,92 @@ func removeInvalidCharFromProperty(key string) string {
 	// be replaced by an "-"
 	return strings.ReplaceAll(key, ".", "_")
 }
+
+func (c *neo4jClient) IngestPackage(ctx context.Context, pkg *model.PkgInputSpec) (*model.Package, error) {
```

Comment:
can we replace `func (c *neo4jClient) registerPackage` in `ingest_testdata.go` with this? This way all the test packages will get ingested into neo4j.

Reply:
That's the plan, once we have all the registration.